### PR TITLE
Collected data list route and view

### PR DESCRIPTION
**Pull request by beamalsky**
## Overview

This PR creates a route and view for a simple list view of collected data, as described in #23 

### Notes

To do:

- [x] Figure out consistent naming for collected data views. @jeancochrane, I'd love your input on this. See: https://github.com/datamade/just-spaces/issues/23#issuecomment-480892552
- [x] Group by survey title. See https://github.com/datamade/just-spaces/issues/23#issuecomment-480905922

## Testing Instructions

 * Start with the db in just-spaces.dump
* Run the site locally and submit a survey! 
* Navigate to http://127.0.0.1:8000/collected-data/

Connects #23

### CODE REVIEW

**Comment by beamalsky** [comment]
Hm, I didn't have any issues applying that new PLDP migration. I've added a new db dump with it, could you try this one out? 

And ee I deleted some data handler urls I didn't think were being used anywhere, but I was wrong! Adding them back should resolve your last comment.

Writing a test here is a great idea—I'm not in that habit and would like to be! Let me give it a try now.

**Comment by jeancochrane** [comment]
Great, new DB dump worked for me! And the missing route is back now. Let me know when you're ready for me to look at the test.

File: tests/conftest.py [hunk 1]
```diff
@@ -102,3 +105,43 @@ def form_element(db, form_entry):
...
+        total=5,
+        survey=survey,
+    )
+
+    return survey_row
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def survey_component(db, row):
```
Comment (jeancochrane) [inline tests/conftest.py]: Are `survey_row` and `survey_component` being used in the test at all, or are you setting them up for the future?

Comment (beamalsky) [reply tests/conftest.py]: Just setting them up for the future! I'll need them to test the next issue (#61)

**Pull request merged**